roles/gitea/tasks/main.yaml

```
@@ -90,9 +107,17 @@
     password: "{{ gitops_password }}"
     body:
       auto_init: true
+      private: true
       name: "{{ gitops_repo_name }}"
     body_format: json
     method: POST
     force_basic_auth: yes
     status_code: 201
   when: repo_data.status == 404
+
+
+- name: Configure GiTea VirtualService
+  community.kubernetes.k8s:
+    state: present
+    template: templates/gitea-vs.yaml.j2
+  when: gitea_create_virtualservice | bool
```

nit: and then maybe over here use (ignore bad syntax)
```suggestion
  when: gitea_create_virtualservice | bool && gitea_use_istio | bool
```